### PR TITLE
feat: revoke user sessions above threshold

### LINKED ISSUE #8730: User can create infinite active sessions which are red from the database on every login
Reference topic:

https://community.nodebb.org/topic/15049/periodical-queries-for-nodebb-sessions

In our case user created over 40k active sessions and with his each login was causing a huge lag as there was a query for every session to database.

I suggested the solution in the previously mentioned topic. Let me know what do you think about it ;-)

Best,
cryptoethic

### PR DESCRIPTION
**Pull request by cryptoethic**
Resolves #8730

### CODE REVIEW

**Comment by nodebb-misty** [comment]
Hello there! :smile:

Our internationalisation project is handled through [our Transifex project](https://www.transifex.com/projects/p/nodebb/). Unfortunately, we cannot accept changes to language files directly via pull request as it would then become out-of-sync with the translations hosted there.

Please re-apply your changes via the Transifex website, and they will be pulled in periodically. If you require the translations sooner, please ping a project maintainer and they can manually pull the translations sooner.

Thank you! :angel:

----

*This is an automated message. If you feel this action was in error, please comment on this pull request so it can be looked at again*

**Comment by julianlam** [comment]
Ah overzealous @nodebb-misty :joy: 

@cryptoethic Can you please only provide changes to `en-GB` please?

File: src/user/auth.js [hunk 1]
```diff
@@ -107,9 +107,18 @@ module.exports = function (User) {
 			return;
 		}
 		await cleanExpiredSessions(uid);
+		await revokeSessionsAboveThreshold(uid, meta.config.maxUserSessions || 10);
```
Comment (julianlam) [inline src/user/auth.js:110]: Instead of setting fallback value here, include it in `install/defaults.json`

Comment (cryptoethic) [reply src/user/auth.js:110]: Will it be sufficient or I need it in two places

**Comment by barisusakli** [comment]
LGTM, thanks

**Comment by cryptoethic** [comment]
Happy to help ;-) 
And many thanks for your suggestions yesterday while debugging

**Pull request merged**